src/ChangesReporting/Output/ConsoleOutputFormatter.php

```diff
@@ -83,15 +88,42 @@ private function reportFileDiffs(array $fileDiffs, bool $absoluteFilePath): void
                 $filePath .= ':' . $firstLineNumber;
             }
 
-            $message = sprintf('<options=bold>%d) %s</>', ++$i, $filePath);
+            $filePathWithUrl = $this->addEditorUrl(
+                $filePath,
+                $fileDiff->getAbsoluteFilePath(),
+                $fileDiff->getRelativeFilePath(),
+                (string) $fileDiff->getFirstLineNumber()
+            );
+
+            $message = sprintf('<options=bold>%d) %s</>', ++$i, $filePathWithUrl);
 
             $this->symfonyStyle->writeln($message);
             $this->symfonyStyle->newLine();
             $this->symfonyStyle->writeln($fileDiff->getDiffConsoleFormatted());
 
             if ($fileDiff->getRectorChanges() !== []) {
+                $relativePathHelper = new ParentDirectoryRelativePathHelper(getcwd());
                 $this->symfonyStyle->writeln('<options=underscore>Applied rules:</>');
-                $this->symfonyStyle->listing($fileDiff->getRectorShortClasses());
+                $appliedRules = [];
+                foreach ($fileDiff->getRectorClasses() as $rectorClass) {
+                    $appliedRuleName = (string) Strings::after($rectorClass, '\\', -1);
+                    $reflectionClass = new ReflectionClass($rectorClass);
+                    $absolutePath = $reflectionClass->getFileName();
+                    if ($absolutePath !== false) {
+                        $relativePath = $relativePathHelper->getRelativePath($absolutePath);
+                        $appliedRuleNameWithUrl = $this->addEditorUrl(
+                            $appliedRuleName,
+                            $absolutePath,
+                            $relativePath,
+                            '0'
+                        );
+                        $appliedRules[] = $appliedRuleNameWithUrl;
+                    } else {
+                        $appliedRules[] = $appliedRuleName;
+                    }
+                }
+
```

Comment:
Thank you for patience :pray:  I was dealing with other PRs and logo redesign last couple of weeks.


This seems like 2 different features. List of rules is only informative. If any, it should lead to detail on https://getrector.com/find-rule, where config and more details are. 

Could you remove this part and stick only with files path?
The rest looks good :+1:

Reply:
PR updated, I removed the links to the rules